splunk_add_on_ucc_framework/commands/rest_builder/global_config_builder_schema.py

```diff
@@ -55,7 +55,7 @@ class GlobalConfigBuilderSchema:
     def __init__(self, global_config: global_config_lib.GlobalConfig):
         self.global_config = global_config
         self._settings_conf_file_names: Set[str] = set()
-        self._configs_conf_file_names: Set[str] = set()
+        self._configs_conf_file_names: List[str] = list()
         self._oauth_conf_file_names: Set[str] = set()
```

Comment:
maybe it's worth to change Set to list also for _settings_conf_file_names and _oauth_conf_file_names to be consistent?

Reply:
thought about leaving rest as maybe worth modifying just this one that does not work but yea lets keep them the same, applied change